packages/mobile/src/App.tsx

```diff
@@ -35,6 +36,8 @@ import { useEnterForeground } from './hooks/useAppState'
 import { WalletConnectProvider } from './screens/wallet-connect'
 import { setLibs } from './services/libs'
 
+allPromisesSettled.shim()
```

Comment:
small thing, but can we add this to the index.js file that imports App.tsx? we have our polyfils set there

Reply:
oh yes def!